test/parallel/test-whatwg-url-tojson.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const URL = require('url').URL;
+
+// Tests below are not from WPT.
+const url = new URL('http://example.org');
```

Comment:
Have to copy/paste at the moment, though I think a tool would definitely helpful.

I think we don't have to update a file as long as the upstream doesn't update a file(at least if it doesn't update the `<script></script>` part. The hope is this way we can see how up-to-date we are with the upstream tests without doing mental conversions from the WPT tests suite to Node.js core test suites.